Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.6.5'
```

Comment:
The version dependency is the same as the [layer.io](https://github.com/layer5io/layer5).

Reply:
Good call.